app/core/tests/evaluation.py

```
@@ -1,60 +1,88 @@
+local_data_path = os.path.join(app_dir, "data", "big_benchmark.csv")
 client = Client(api_key=api_key)
 
-# Dataset configuration
-dataset_name = "smaller_benchmark_temporal"
+dataset_name = "test_metabot"
+
+# Check if the dataset exists in the current user's workspace
+try:
+    client.read_dataset(dataset_name=dataset_name)
+    logger.info(f"Dataset '{dataset_name}' found in LangSmith workspace.")
+except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)